app/assets/stylesheets/hawkers.scss

```diff
@@ -55,6 +55,10 @@
 	width: 300px;
 }
 
+.photo {
```

Comment:
need to use a more explicit classname as .photo may affect other images in the app called photo as well